src/tests/stdlib/memcmp_test.c

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file
+ * @brief Test unit for string/memcmp
+ *
+ * @date March 18,2020
+ * @author Sha Ahammed Roze
+ */
+
+ #include <embox/test.h>
+ #include <strings.h>
+
+ EMBOX_TEST_SUITE("Test suite for memcmp()");
+
+TEST_CASE("Test that buf1 > buf2") {
+    int n;
+    char buf1[] = "SWgaOtP43df2";
+    char buf2[] = "SWGaOTP12df0";
+ 
+    n = memcmp(buf1, buf2, sizeof(buf1));
+    test_assert(n > 0);
```

Comment:
test_assert_equal?

Reply:
test_assert_equal should be true? I though it automatically checked that?

Reply:
@anton-bondarev doesnt test pass if the condition inside test_asset is true? why do i need test_assert_equal?

Reply:
@sha-ahammed 
I mean you can use strict equality
test_assert(n > 0); -> test_assert(n == ..);

Reply:
ok, it's good